Zend/zend_types.h

```diff
@@ -829,6 +830,13 @@ static zend_always_inline uint32_t zval_gc_info(uint32_t gc_type_info) {
 
 #define OBJ_FLAGS(obj)              GC_FLAGS(obj)
 
+/* object extra flags (zend_object.flags) */
+
+#define IS_OBJ_LAZY                 (1U<<31) /* Virtual proxy or uninitialized Ghost */
+#define IS_OBJ_LAZY_PROXY           (1U<<30) /* Virtual proxy (may be initialized) */
+
+#define OBJ_EXTRA_FLAGS(obj)		((obj)->flags)
```

Comment:
It's mainly because there is also a `OBJ_FLAGS()` (which expands to `GC_FLAGS()`). Unfortunately every bit in `OBJ_FLAGS()` is already used, that's why we need a new field.

I've renamed the field to `extra_flags`.